src/VirtoCommerce.DemoCustomerSegmentsModule.Core/Models/DemoCustomerSegment.cs

```diff
@@ -15,6 +15,8 @@ public DemoCustomerSegment()
 
         public string Description { get; set; }
 
+        public string UserGroup { get; set; }
```

Comment:
[StringLength(64)]
https://github.com/VirtoCommerce/vc-module-customer/blob/ff7dd139fcbb8a9ead90c988d141d5dadec7cd0e/src/VirtoCommerce.CustomerModule.Data/Model/MemberGroupEntity.cs#L9

Reply:
Oh, sorry, it's about Data model, of course